tests/test_ssl.py

```diff
@@ -101,6 +101,21 @@
 skip_if_py3 = pytest.mark.skipif(PY3, reason="Python 2 only")
 
 
+def socket_any_family():
+    try:
+        return socket(AF_INET)
+    except error:
```

Comment:
I'm sure I'll regret asking, is there any way for us to narrow the exception down so we don't handle other failures?

Reply:
Yes, this should have an `errno` attribute, let's check that it's the right one.

Reply:
I take it Python on Windows will try to map the errno and such?

Reply:
I think so. I've definitely written a lot of code that does `.errno == ENOENT` and no one has complained 😬

Reply:
Done.